content/contributing-code/usability/contents.lr

```diff
@@ -0,0 +1,60 @@
+
+**Schedule tests.** Promote with your social users, newsletter subscribers, announce it on your site, or get the word out some other way. We use Calendly for scheduling these, and we include a number of basic questions during the session sign-up process, to help us better prepare for tests. (See an example sign-up [here](https://calendly.com/cc-product-design/user-interview-30) ). Our typical requirements for participants are high speed internet access, a computer with a microphone, ability to screen share, and request for consent form to be signed. For anything new you want to get feedback on, it’s advisable to talk to at least three users.
+
+**Rehearse.** Before your first session, simulate a whole session, read the script, complete the task(s), and answer the final questions you plan to ask. Think about if there are additional things you want to ask. Be ready, knowing what you want to say and ask.
+
+**Carry out the test.** During the session, have your script open on one window, and a video call with screen share open on another window. We use Google Meet (Hangouts) for our video calls.
+
+Have a notes document prepared, or better yet, enlist someone else to join the call to take notes. If (and only if) your participant has consented to screen share being recorded, be ready with software that records the screen. We often use QuickTime (if on a Mac), it is simple and free. Another option is [OBS](https://obsproject.com/), which is free, open source, and cross platform.
+
+Remember that the point of the test is to ask questions and get feedback. It is not to educate, give guidance, or explain. Whether you are running a usability test, remember that you want  to see what the user will do to complete the task when no one is around to help them. Ask them to think out loud, so that you can better understand what is clear and what is confusing, and pay close attention to how the user interacts with the prototype. If something related to your research purpose catches your attention, dive into it by asking why.
+
+**Thank the user.** At the end of the test, remember to thank the user. If you have the budget for it, we recommend signaling gratitude with a gift card, or other reward. Members of your community may politely decline this offer, because their reward is the ability to contribute to improving your product.
+
+**Review your findings.** Spend time with your notes, rewatching the screen recording, and deeply thinking about what you have learned.
+
+We strongly recommend reading [Rocket Surgery Made Easy by Steve Krug](http://stevekrug.com/rsme.html ). The methods we employ at CC are to a large extent based on the guidance in that book. Have you read something fantastic about usability testing? Join us in the #cc-usability channel in the [Creative Commons Slack](https://slack-signup.creativecommons.org/) and share your resources!
```

Comment:
Please put the Slack channel name in back ticks, like you did in the "Participate" section when it was referenced.